### PR TITLE
Schema api enhancements

### LINKED ISSUE #250: Enhance /indexer/schema API endpoint
**Is your feature request related to a problem? Please describe.**

- /API/v1/indexer/schema returns all indexer details which is useful to create a list of supported indexers but has some issues.

- `"infoLink"` points to `"https://wikijs.servarr.com/prowlarr/supported#{indexer-name}"`
  - Cardigann Indexers `"infoLink"` point to `"infoLink": "https://wikijs.servarr.com/prowlarr/supported#cardigann"`
- many indexers are missing a descripton

**Describe the solution you'd like**

- Change all `"infoLink"` to point to `"https://wikijs.servarr.com/prowlarr/supported-indexers`
- Enhance all indexers to include descriptions

**Describe alternatives you've considered**

- n/a

**Additional context**

Enhancements will allow easier automation of outputting a table of supported indexers.

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
- Fixes cardigann indexers missing description from API
- Fixes info link for indexers

#### Todos
- Tests
- Translation Keys
-  Wiki Updates

#### Issues Fixed or Closed by this PR

* Fixes #250

### CODE REVIEW

File: src/Prowlarr.Api.V1/ProviderResource.cs [hunk 1]
```diff
@@ -39,8 +39,8 @@ public virtual TProviderResource ToResource(TProviderDefinition definition)
                 Fields = SchemaBuilder.ToSchema(definition.Settings),
 
                 //prowlarr/supported#{0} are custom build redirect pages; if passing a new var, create a new redirect
-                InfoLink = string.Format("https://wikijs.servarr.com/prowlarr/supported#{0}",
-                    definition.Implementation.ToLower())
+                InfoLink = string.Format("https://wikijs.servarr.com/prowlarr/supported-indexers#{0}",
```
Comment (RobinDadswell) [inline src/Prowlarr.Api.V1/ProviderResource.cs:42]: Does this link actually work (e.g. the #torrentgalaxy) as they aren't headers...

**Review by RobinDadswell — COMMENTED** [review]
Just the one question

File: src/Prowlarr.Api.V1/ProviderResource.cs (hunk 1 above)
Comment (bakerboy448) [inline src/Prowlarr.Api.V1/ProviderResource.cs:42]: Need to Revert this change as it applies to all.

need to conditionalize this to be for indexers only

Comment (bakerboy448) [reply src/Prowlarr.Api.V1/ProviderResource.cs:42]: yeah still need to figure that out on the wiki since we're using a table... also need to make this conditional to only apply to indexers; not all implementations

**Review by Qstick — APPROVED** [review]
